src/one/nio/serial/GeneratedSerializer.java

```diff
@@ -53,7 +60,13 @@
         }
 
         checkFieldTypes();
+        for (FieldDescriptor fd : fds) {
+            if (!generationInProcess.contains(fd.type().resolve())) {
+                Repository.get(fd.type().resolve());
```

Comment:
I'm afraid, this is not that simple. The type of a field can be an interface, an abstract class, super class, or just `java.lang.Object`. Generating serializers for these types does not make much sense. The actual type of the field value may differ. So, the common case is not covered; does it really make sense to care about special cases, when actual field type is equal to the declared type?